### PR TITLE
Add translated strings

### PR DESCRIPTION
**Pull request by marinaaisa**
Bug/issue #108904481, if applicable: 

## Summary

Add translated strings

## Dependencies

NA

## Testing

Steps:
1. Add the following to `theme-settings.json` file at public folder
```json
{
  "features": {
    "docs": {
      "i18n": {
        "enable": true
      }
    }
  }
}
```
3. Run `npm run serve`
4. Go to the bottom of the page and change the language in the locale selector to Chinese and Japanese
5. Assert that strings are correct

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

**Comment by mportiz08** [comment]
> Japanese translation of "view-in" and “continue-viewing” are wrong, I'll wait for the translators to get back to me.
> Don't merge this PR yet.

Can you make this a draft PR until then so we don't accidentally merge this before it's ready?

**Comment by marinaaisa** [comment]
@swift-ci test

**Comment by marinaaisa** [comment]
@swift-ci test

File: src/lang/locales/zh-CN.json [hunk 1]
```diff
@@ -101,31 +110,31 @@
...
-    "search-symbols": "搜索符号",
+    "search-symbols": "在 {technology} 搜索符号",
     "suggested-tags": "建议标签",
     "selected-tags": "所选标签",
     "add-tag": "添加标签",
     "tag-select-remove": "标签。选择以从列表中移除。",
     "navigate": "若要导航符号，请按下上箭头、下箭头、左箭头或右箭头。",
-    "siblings-label": "内含 {number-siblings} 个符号（共 {total-siblings} 个）{parent-siblings}",
-    "parent-label": "@:filter.siblings-label 包含 1 个符号 | @:filter.siblings-label 包含 {number-parent} 个符号",
+    "siblings-label": "{Parent-siblings} 内含 {number-siblings} 个符号（共 {total-siblings} 个）",
```
Comment (mportiz08) [inline src/lang/locales/zh-CN.json:130]:
```suggestion
    "siblings-label": "{parent-siblings} 内含 {number-siblings} 个符号（共 {total-siblings} 个）",
```

**Comment by mportiz08** [comment]
@swift-ci test

**Review by mportiz08 — APPROVED** [review]

**Pull request merged**